src/test/java/core/basesyntax/ConcatenatedWordsTest.java

```diff
@@ -0,0 +1,121 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ConcatenatedWordsTest {
+    private final static String LONGEST_CONCATENATED_WORD = "ratcatdogcat";
+    private final static String SECOND_LONGEST_CONCATENATED_WORD_WHEN_SAME_LENGTH = "catdogcatrat";
+    private final static String SECOND_LONGEST_CONCATENATED_WORD = "catsdogcats";
+    private final static String NO_CONCATENATED_WORD = "";
+    private final static Integer TOTAL_COUNT_OF_CONCATENATED_WORDS = 3;
+    private final static String[] ARRAY_OF_WORDS = {"cat", "cats", "catsdogcats",
+            "dog", "hippopotamuses", "rat",
+            "ratcatdogcat", "dogcatsdog"};
+    private final static String[] ARRAY_OF_NOCONCATENATED_WORDS = {"cat", "cats", "dog",
+            "hippopotamuses", "rat",};
+    private final static String[] ARRAY_WITH_CONCATENATED_WORDS_OF_SAME_LENGTH =
+            {"cat", "cats", "catsdogcats",
+                    "ratcatdogcat", "dog", "hippopotamuses",
+                    "rat", "ratcatdogcat", "dogcatsdog"};
+    private List<String> listOfWords;
+    private List<String> emptyListOfWords;
+    private List<String> listWithNoConcatenatedWords;
+    private List<String> listOfWordsWhereTheSameLengthOfConcatenatedWord;
+    private ConcatenatedWords concatenatedWords;
+
+    @Before
+    public void init() {
+        listOfWords = new ArrayList<>();
+        Collections.addAll(listOfWords, ARRAY_OF_WORDS);
+
+        emptyListOfWords = new ArrayList<>();
+
+        listWithNoConcatenatedWords = new ArrayList<>();
```

Comment:
```suggestion
        listWithNoConcatenatedWords = List.copyOf(ARRAY_OF_NOCONCATENATED_WORDS);
```
тоді ти можеш прибрати Collections.addAll(li...

Reply:
List.copyOf(Collection c) приймає на вхід колекцію, a List.of() набір елементів, я проініціалізував Arrays.asList()